app/presenters/submission_presenter.rb

```diff
@@ -33,4 +34,13 @@ def previous_to_rate
   def submission
     __getobj__
   end
+
+  def average(rates)
```

Comment:
You don't need to pass rates here and I think passing them is kind of confusing, because you already have `@rates` in submission